### PR TITLE
Fix broken links in CONTRIBUTING.md

### LINKED ISSUE #17062: In contributing.md, clicking the 'contribution guidelines' link does not take to any contribution guidelines.
#### Challenge Name

https://github.com/freeCodeCamp/freeCodeCamp/blob/staging/CONTRIBUTING.md
and 
https://github.com/freeCodeCamp/freeCodeCamp/blob/staging/CONTRIBUTING.md#contribution-guidelines

#### Issue Description

There are two links associated with reading 'contribution guidelines'. When I click the link, '#contribution-guidelines' is added to the link (posted above) however, the page does not jump to any guidelines. There also doesn't seem to be anything on the page titled as or associated with 'contribution guidelines'. If the link just means that the general `contributing.md` is the 'contributing guidelines', then it could possibly be misleading (it was to me, personally).

#### Browser Information

* Browser Name, Version: Mozilla Firefox (59.0.1) 
* Operating System: Ubuntu 16.04
* Mobile, Desktop, or Tablet: Desktop

#### Your Code

```js



```

### PR DESCRIPTION
**Pull request by Dipeshwagle**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [x] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes #17062

#### Description

Links associated with 'contribution guidelines' were broken, due to change from 'Contribution Guidelines header' to 'Table of Contents' in [this](https://github.com/freeCodeCamp/freeCodeCamp/commit/6bc521815aa68e3296935dec3f292d64dba65e6c) commit. Fixed links so it goes to to 'Table of Contents'.

### CODE REVIEW

**Comment by rohanBagchi** [comment]
I think renaming the `Table of Contents` to `Contribution Guidelines` would make more sense. 
The list on top does say :
Follow the **Contribution Guidelines** to start working on the issue.
Personally I would expect a list with a similar title.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ We welcome pull requests from freeCodeCamp campers (our students) and seasoned J
 
 2. Let us know you are working on it by posting a comment on the issue.
 
-3. Follow the [Contribution Guidelines](#contribution-guidelines) to start working on the issue.
+3. Follow the [Contribution Guidelines](#table-of-contents) to start working on the issue.
```
Comment (raisedadead) [inline CONTRIBUTING.md]: Lets just get rid of this link. I think its not required.

```diff
- 3. Follow the [Contribution Guidelines](#table-of-contents) to start working on the issue.
+ 3. Follow the instructions in this guide to start working on the issue.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -436,7 +436,7 @@ Instance of freeCodeCamp](#maintaining-your-fork).
 4.  Create a branch off of `staging` with git: `git checkout -B
     branch/name-here` **Note:** Branch naming is important. Use a name like
     `fix/short-fix-description` or `feature/short-feature-description`. Review
-     the [Contribution Guidelines](#contribution-guidelines) for more detail.
+     the [Contribution Guidelines](#table-of-contents) for more detail.
```
Comment (raisedadead) [inline CONTRIBUTING.md]:
```diff
- .... Review the [Contribution Guidelines](#table-of-contents) for more detail.
```

Comment (raisedadead) [reply CONTRIBUTING.md]: Remove the above sentence.

**Review by raisedadead — CHANGES_REQUESTED** [review]

**Pull request merged**